common/logisticspipes/gui/guidebook/GuiGuideBookTabButton.java

```diff
@@ -0,0 +1,52 @@
+package logisticspipes.gui.guidebook;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+
+import lombok.Setter;
+
+import logisticspipes.LPConstants;
+
+public class GuiGuideBookTabButton extends GuiButton {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
```

Comment:
I have seen `GUI_BOOK_TEXTURE` several times now. I think initializing it once and using the same constant over several classes would be better?

Reply:
Also added to the Glo Board